.github/workflows/download_pipeline.yml

```
@@ -0,0 +1,142 @@
+        run: nextflow run ./${{ needs.configure.outputs.REPOTITLE_LOWERCASE }}/$( sed 's/\W/_/g' <<< ${{ needs.configure.outputs.REPO_BRANCH }}) -profile test,singularity --outdir ./results
+
+      - name: Count the downloaded number of container images
+        id: count_afterwards
+        run: |
+          image_count=$(ls -1 ./singularity_container_images | wc -l | xargs)
+          echo "Post-pipeline run container image count: $image_count"
+          echo "IMAGE_COUNT_AFTER=$image_count" >> "$GITHUB_OUTPUT"
+
+      - name: Compare container image counts
+        id: count_comparison
+        run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2028:info:5:8: echo may not expand escape sequences. Use printf

(shellcheck)